supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/Delocalization.scala

```diff
@@ -17,27 +17,30 @@ import scala.collection.JavaConverters._
 import scala.concurrent.duration._
 object Delocalization {
   private val logsRoot = "/google/logs"
+  val plainTextContentType = Option(ContentTypes.`text/plain(UTF-8)`)
 }
 
 trait Delocalization {
 
   private def aggregatedLog = s"$logsRoot/output"
 
-  private def delocalizeLogsAction(gcsLogPath: String, projectId: String) = {
-    // To re-enable requester pays, this need to be added back: "-u", projectId
-    gsutilAsText("-m", "cp", "-r", "/google/logs", gcsLogPath)(flags = List(ActionFlag.AlwaysRun))
+  private def delocalizeLogsAction(gcsLogPath: Path, projectId: String) = {
```

Comment:
Forgot to mention: This method accepts a `projectId` but is not using it. Are there plans to somehow use the `projectId` in the future? If so I would re-add a comment. If not, then I would remove this unwired parameter.

Reply:
It's a leftover, will remove